contrib/net/http/roundtripper.go

```diff
@@ -58,8 +59,8 @@ func (rt *roundTripper) RoundTrip(req *http.Request) (res *http.Response, err er
 		span.SetTag(ext.Error, err)
 	} else {
 		span.SetTag(ext.HTTPCode, strconv.Itoa(res.StatusCode))
-		// treat 5XX as errors
-		if res.StatusCode/100 == 5 {
+		// check for 5XX server codes
+		if globalconfig.IsHTTPServerError(res.StatusCode) {
```

Comment:
Isn't this a client? I'm a bit confused here because previously we were counting `5xx` as errors... Wasn't that wrong?

Reply:
Apparently in our roundtripper [tests](https://github.com/DataDog/dd-trace-go/blob/fe5a1d5bc16973c7815a971dd8a1cb0811612b76/contrib/net/http/roundtripper_test.go#L71), we do check if a newly created server can send an errored request to the client, which contains the `RoundTripper` struct.
Based on my understanding of the documentation of [Client](https://golang.org/pkg/net/http/), we would need to check for server codes in order to ensure that the client's response result from `RoundTrip` is an error when communicating to the server.